modules/administration/pages/troubleshooting/tshoot-corruptrepo.adoc

```diff
@@ -27,8 +27,8 @@ With an new repository data file, updates should operate as expected.
 .Procedure: Resolving Corrupt Repository Data
 . Remove all files from [path]``/var/cache/rhn/repodata/<channel-label>``.
     If you do not know the channel label, you can find it in the {productname} {webui}, by navigating to menu:Software[Channels > Channel Label].
-. Regenerate the file from the command line:
+. On the container host, regenerate the file from the command line:
```

Comment:
Yes, that's misleading. Trying to find a short but better phrase:
```suggestion
. On the container host, execute from the command line to regenerate the file in the container:
```

Reply:
A quick note; you're missing a noun in your suggestion. Execute what?

I'd suggest something like: `... execute the following command from the command line to ...`

Or alternatively, `On the container host, regenerate the file in the container from the command line:`

(Sorry, a passer-by comment :) )